knotx-performance-tests/config-knotx/author-config.json

```diff
@@ -0,0 +1,182 @@
+{
```

Comment:
Do we need those 2 configurations? Are they used? What is `publish` and what is `author` config?